server/src/main/java/com/coincoinche/engine/CoincheGame.java

```diff
@@ -0,0 +1,66 @@
+package com.coincoinche.engine;
+
+import com.coincoinche.engine.game.GameResult;
+import com.coincoinche.engine.game.RedBlueRotatingPlayersGame;
+import com.coincoinche.engine.teams.Player;
+import com.coincoinche.engine.teams.Team;
+
+/** Implementation of a coinche game. */
+public class CoincheGame extends RedBlueRotatingPlayersGame<Player> {
+
+  private static final int WINNING_POINTS = 1000;
+  // TODO nockty: consider DI for this
```

Comment:
DI stands for dependency injection. Basically, `CoincheGame` currently initializes its `CoincheGameRound` dependency, but it could be better to have this initialization decoupled from `CoincheGame`, by having another class injecting the `CoincheGameRound` dependency (_e.g._ a factory).
Not sure it'll be useful though. Need to think about it.